src/common/showmsg.cpp

```diff
@@ -646,6 +646,9 @@ int _vShowMessage(MSGTYPE flag, const std::string& string)
         case MSG_ACTION: // Bright White  (mostly useless "player did this" info)
             strcat(prefix, CL_WHITE"[Action Info]" CL_RESET);
             break;
+        case MSG_EXPLOIT: // Bright Red (Detected a likely exploit)
+            strcat(prefix, CL_RED"[Error]" CL_RESET);
```

Comment:
Should this be `Exploit` instead of `Error`?

Reply:
good catch